### ISSUE TITLE
Failed to to install pak

### ISSUE THREAD

**Issue by mhurtado13** [r-lib/rig#226]
[ERROR] Failed to to install pak for R 4.1.3: Failed to find user HOME

**Comment by gaborcsardi** [comment]
I am afraid you'll have to tell us what kind of system this is.

**Comment by mhurtado13** [comment]
Hello, thanks for your quick answer! 

My system is Ubuntu 22.04 and this happens when I ran "rig add **version**"

For now it's working when I put "rig add **version** --without pak" but I just wanted to know if there is a way to solve it cause I didn't find information about this.

**Comment by gaborcsardi** [comment]
What is the output of these?
```
id 
```

```
echo $HOME
```

Can you show the full output of 
```
rig add release
```
?

**Comment by mhurtado13** [comment]
![image](https://github.com/r-lib/rig/assets/37853385/2ba81a9a-583d-4e02-aa31-60ec9cd5e724)

![image](https://github.com/r-lib/rig/assets/37853385/9c24a97f-2963-4fc7-b594-e88397188ff3)

**Comment by gaborcsardi** [comment]
Is it possible that your `sudo` is configured not to keep environment variables. E.g. can you copy-paste the output if these?
```
export FOO=bar
sudo R -q -e 'Sys.getenv("FOO")'
sudo -E R -q -e 'Sys.getenv("FOO")'
```

**Comment by mhurtado13** [comment]
![image](https://github.com/r-lib/rig/assets/37853385/0489e6e1-bcc2-4ecf-a8ea-8de1696cc74e)

**Comment by gaborcsardi** [comment]
I am running out of ideas, but what's the output of this?
```
sudo R -q -e 'Sys.getenv("HOME")'
sudo -E R -q -e 'Sys.getenv("HOME")'
```

**Comment by mhurtado13** [comment]
![image](https://github.com/r-lib/rig/assets/37853385/5af69041-75c7-4600-ae57-37320745eb6e)

**Comment by gaborcsardi** [comment]
Do you have some special sudo installed or configured? E.g. did you change `/etc/sudoers`?

What's the output of these?
```
which -a sudo
dpkg -l | grep sudo
```

**Comment by mhurtado13** [comment]
No that I know, this computer belong to the center where I work so there is a super user who administrates it. However, I have sudo privileges (can install with sudo normally) Not sure if it is related though

![image](https://github.com/r-lib/rig/assets/37853385/89ecb7e7-2255-40d6-b36b-fdeaa63c3078)

**Comment by gaborcsardi** [comment]
Yes, I suspect that this is an interaction of how your sudo is set up and how rig works. I would be happy to fix it in rig, but I don't really know what is going wrong.

Nevertheless, in the meanwhile you can use `--without-pak`, and if you want to install pak later, you can do that with
```
source("https://pak.r-lib.org/install.R")
```

**Comment by mhurtado13** [comment]
Yes, I installed it on that way (without-pak) I know it is not a big issue cause you always can install pak after from R so no big deal. Thanks anyway for your fast replies and for your great work regarding rig! Amazing tool 👍🏽

**Comment by gaborcsardi** [comment]
I'll leave this issue open until the next rig release, and before the next release I'll some more debug logging to this part, so we'll have a chance to see what's going wrong.

**Comment by llrs** [comment]
I have the same trouble, in my case my user name is tied to the Microsoft account from the work place and it contains an @ (which I thought could be a problem):

```
lrevilla@irsicaixa.es@mod2-m49:~$ id
uid=868805708(lrevilla@irsicaixa.es) gid=868803355(gs_branderlab_rw@irsicaixa.es) groups=868803355(gs_branderlab_rw@irsicaixa.es),27(sudo),999(shiny),1002(work),1003(shiny-apps),868800513(usuarios del dominio@irsicaixa.es)
lrevilla@irsicaixa.es@mod2-m49:~$ pwd
/home/lrevilla@irsicaixa.es
```

All the other info is similar to what is reported by Marcelo Hurtado.

**Comment by gaborcsardi** [comment]
I believe that this is already fixed by falling back to using `$HOME` when the user does not have an `/etc/passwd` entry. Please open a new issue if you still have problems.
